### ISSUE TITLE
Possible to Update with EAN?

### ISSUE THREAD

**Issue by muellernm** [dweeves/magmi-git#228]
Hello, we have a list with products only with EAN, but in SHop we have SKU and EAN. So is it possible to update with a CSV which only have the EAN?

**Comment by dweeves** [comment]
Yes, using the "SKU FInder" plugin. However you should set "EAN" attribute
"unique" in magento admin for this plugin to work.

2015-03-30 8:29 GMT+02:00 Wolfgang Müller <notifications@github.com>:
